test/unit/controllers/PdfDownloadControllerSpec.scala

```diff
@@ -71,6 +71,11 @@ class PdfDownloadControllerSpec extends ControllerSpecBase with MockitoSugar {
     when(fileService.getAttachmentMetadata(any[Case])(any[HeaderCarrier])).thenReturn(successful(Seq.empty))
   }
 
+  private def givenTheFileServiceFindsLetterOfAuthority(): Unit = {
```

Comment:
The method name suggests something was found but the code is returning `None`. Is that correct?